iris_grib/_load_convert.py

```diff
@@ -1510,14 +1510,14 @@ def hybrid_factories(section, metadata):
                                 {'long_name': 'sigma'},
                                 Reference('ref_surface_pressure')]
 
-            # Create the level pressure scalar coordinate.
+            # Create the level height/pressure scalar coordinate.
             pv = section['pv']
-            offset = scaledValue - 1
+            offset = scaledValue
```

Comment:
Maybe worth an extra comment here also, about 'scaledValue' being 1-based, or "1..n_levels".
And/or maybe rename the variable 'level_number' ??

Reply:
I don't agree with renaming the variable `level_number` as offset is meant to be the offset within the the pv array, see line 1520 where the offset is different

Reply:
Sorry, I meant that you could rename ***`scaledValue`*** as `level_number`.  I.E. what you get as the principal coordinate value is in this case a level number, i.e. a 1..N value.

I will check out your other changes..